contributors/devel/api_changes.md

```diff
@@ -107,24 +109,34 @@ does not add a new required field)
      * which fields are required and which are not
      * mutable fields do not become immutable
      * valid values do not become invalid
+     * explicitly invalid values do not become valid
 
 Put another way:
 
-1. Any API call (e.g. a structure POSTed to a REST endpoint) that worked before
-your change must work the same after your change.
-2. Any API call that uses your change must not cause problems (e.g. crash or
-degrade behavior) when issued against servers that do not include your change.
-3. It must be possible to round-trip your change (convert to different API
+1. Any API call (e.g. a structure POSTed to a REST endpoint) that succeeded
+before your change must succeed after your change.
+2. Any API call that does not use your change must behave the same as it did
+before your change.
+3. Any API call that uses your change must not cause problems (e.g. crash or
+degrade behavior) when issued against an API servers that does not include your
+change.
+4. It must be possible to round-trip your change (convert to different API
 versions and back) with no loss of information.
-4. Existing clients need not be aware of your change in order for them to
-continue to function as they did previously, even when your change is utilized.
+5. Existing clients need not be aware of your change in order for them to
+continue to function as they did previously, even when your change is in use.
+6. It must be possible to rollback to a previous version of API server that
```

Comment:
How many versions are required to be roll-backable? I think just 1, but we should say that?